tests/python/rest_api/test_analytics.py

```diff
@@ -35,11 +35,17 @@ def _test_cannot_see(self, user):
         assert response.status_code == HTTPStatus.FORBIDDEN
 
     @pytest.mark.parametrize(
-        "privilege, is_allow",
-        [("admin", True), ("business", True), ("worker", False), ("user", False)],
+        "conditions, is_allow",
+        [
+            (dict(privilege="admin"), True),
+            (dict(privilege="business"), True),
+            (dict(privilege="worker", has_analytics_access=False), False),
```

Comment:
One more minor comment that can be fixed later:
If I remember correctly, we do not use a `worker` group anywhere right now, but based on the current rules, a user with a `worker` group should have analytics access if `has_analytics_access` is set to True. I'm not sure if this will be a real situation in the future, but tests should cover all possible cases.

Reply:
added test for worker with has_analytics_access=True